### ISSUE TITLE
Do not disable mounted looting if CH is lower in the load order than SHSE

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#122]
Nexus request
Q.
> I would rather just turn off CH looting and use your mod. If that's a pain, could you tell me how to turn off your CH detection/switch off function? Right now, I have to use CH's looting mod, and it's not as robust as yours.
A.
> This is an edge case for interop with one mod, and I'd prefer not to include in the MCM which affects every user.
Would it work for you if I ignore the CH mounted check if SHSE is higher in the load order than CH? I believe you can relocate SHSE to wherever you like, within reason (e.g. don't put it above Dawnguard.esm, for example).

**Comment by SteveTownsend** [comment]
Test evidence. Removed ESM flag to allow flexible load ordering of SHSE.
```
J:\GitHub\SmartHarvestSE\src\Data\LoadOrder.cpp(77): [MESSAGE] {3936} (2020-07-12 09:37:19 AM) SmartHarvestSE.esp has FormID mask 0xfe000000, priority 6
J:\GitHub\SmartHarvestSE\src\Data\LoadOrder.cpp(77): [MESSAGE] {3936} (2020-07-12 09:37:19 AM) Convenient Horses.esp has FormID mask 0x09000000, priority 10
J:\GitHub\SmartHarvestSE\src\WorldState\PlayerState.cpp(268): [MESSAGE] {3936} (2020-07-12 09:37:22 AM) SHSE handles looting while mounted: Convenient Horses loads after SHSE
```
```
J:\GitHub\SmartHarvestSE\src\Data\LoadOrder.cpp(77): [MESSAGE] {34872} (2020-07-12 09:54:33 AM) Convenient Horses.esp has FormID mask 0x09000000, priority 9
J:\GitHub\SmartHarvestSE\src\Data\LoadOrder.cpp(77): [MESSAGE] {34872} (2020-07-12 09:54:33 AM) SmartHarvestSE.esp has FormID mask 0xfe000000, priority 10
J:\GitHub\SmartHarvestSE\src\WorldState\PlayerState.cpp(263): [MESSAGE] {34872} (2020-07-12 09:54:35 AM) Block looting while mounted: Convenient Horses loads before SHSE
```

**Comment by SteveTownsend** [comment]
Fixed for v1.7.1.0
